src/components/pages/admin.js

```diff
@@ -94,7 +94,7 @@ function Admin({ PageTemplate }) {
   return (
     <PageTemplate>
       <div className="AdminPage">
-        <h1>{t('pageNames:admin')}</h1>
+        <h1 className="u-page-title">{t('pageNames:admin')}</h1>
```

Comment:
We also have a new page merged in the meantime with a title: `comparison` :grinning:

Reply:
I actually made a mental note on that and the forgot, thanks for reminding me!
Updated.

Reply:
:+1: